karma.conf.js

```diff
@@ -12,5 +12,11 @@ export default function configKarma(karma : Object) {
         webpack:  WEBPACK_CONFIG_TEST
     });
 
+    karmaConfig.coverageReporter = {
```

Comment:
@amyegan thanks for working on these badges and standardizing code coverage reporting! 

What do you think about adding this reporter to grumbler-scripts instead? It seems like every repo using codecov needs the lcov reporter type and it's currently missing from the shared karma config in grumbler-scripts. I'm thinking we could solve this for karma like so:

```js
    // every repo's karma script passes `coverage: true`
    const karmaConfig = getKarmaConfig(karma, {
        basePath:  __dirname,
        testDir:   'test',
        windowDir: 'test/integration/windows',
        entry:     'test/integration/index.js',
        webpack:   WEBPACK_CONFIG_TEST,
        // use the existing coverage option
        coverage:  true  
    });
```

To make `coverage: true` work with codecov we'd need to add this lcov reporter here: https://github.com/krakenjs/grumbler-scripts/blob/master/config/karma.conf.js#L152-L172. Would you be up for making this change in grumbler-scripts?